tools/ci/run_tc.py

```diff
@@ -102,8 +102,12 @@ def get_parser():
                    help="Install web-platform.test certificates to UA store")
     p.add_argument("--no-install-certificates", action="store_false", default=None,
                    help="Don't install web-platform.test certificates to UA store")
-    p.add_argument("--rev",
-                   help="Revision that the task_head ref is expected to point to")
+    p.add_argument("--ref",
+                   help="Ref that's should be run")
```

Comment:
```suggestion
                   help="Ref that should be run")
```

But also I would suggest something like: `Git ref of the change Taskcluster should be run for`. Maybe that's just my preference for verbosity :D